js/flightlog_index.js

```diff
@@ -261,7 +290,16 @@ function FlightLogIndex(logData) {
                     resultIndex.avgThrottle[j] = sourceIndex.avgThrottle[j] - 1000;
                 }
             }
-            
+            if (sourceIndex.maxRC.length > 0) {
+                for (j = 0; j < sourceIndex.maxRC.length; j++) {
+                    resultIndex.maxRC[j] = sourceIndex.maxRC[j] * 20 - 1000;
+                }
+            }
+            if (sourceIndex.maxMotorDiff.length > 0) {
+                for (j = 0; j < sourceIndex.maxRC.length; j++) {
```

Comment:
```suggestion
                for (j = 0; j < sourceIndex.maxMotorDiff.length; j++) {
```

Reply:
Thanx - they should be the same length but it should still be proper copypasta!